integration-testing/smokeTest/build.gradle

```diff
@@ -40,4 +41,12 @@ kotlin {
             }
         }
     }
+    targets {
+        configure([]) {
+            tasks.getByName(compilations.main.compileKotlinTaskName).kotlinOptions {
+                jvmTarget = "1.8"
```

Comment:
Isn't it enabled by default?

Reply:
@woainikk could you please eleborate why we need to set jvmTarget here?